### ISSUE TITLE
Question: Android 12 can background image cover all background or is it only rendered as icon?

### ISSUE THREAD

**Issue by adrianvintu** [jonbhanson/flutter_native_splash#318]
On my Samsung Galaxy S10 with Android 10, setting the image to a big image, like `flutter_native_splash-master\example\assets\logo_lockup_flutter_vertical.png` it only shows it as a medalion circle.

At this address https://developer.android.com/guide/topics/ui/splash-screen I see the splash is a medalion circle.

Is it possible to have a big background like this one `logo_lockup_flutter_vertical.png`  covering all background, instead of being rendered as a medalion icon, on Android 12?

In other words:
Using the documentation here https://developer.android.com/guide/topics/ui/splash-screen
```
4 The window background consists of a single opaque color. If the window background is set and is a plain color, it is used by default if the attribute is not set.
```
Can this be an image as well, or only a color?

**Comment by jonbhanson** [comment]
> Is it possible to have a big background like this one `logo_lockup_flutter_vertical.png` covering all background, instead of being rendered as a medalion icon, on Android 12?

What I understand from that documentation is that Android 12 only supports a background color.  A background image is not supported.

**Comment by adrianvintu** [comment]
@jonbhanson would you be so kind as to add this to the documentation? I spent a lot of time debugging this issue.
